apps/spruce/src/pages/projectSettings/tabs/PluginsTab/getFormSchema.tsx

```diff
@@ -197,7 +197,7 @@ export const getFormSchema = (
       perfEnabled: {
         "ui:widget": widgets.RadioBoxWidget,
         "ui:disabled": isRepo,
-        "ui:description": `Enable the performance plugin (this requires the project to have matching ID and identifier). ${isRepo && "This setting is disabled at the repo level."}`,
+        "ui:description": `Enable the performance plugin (this requires the project to have matching ID and identifier). ${isRepo ? "This setting is disabled at the repo level." : ""}`,
```

Comment:
I wonder if we should omit the first sentence at the repo level? Since there's no way at all they can enable it -- up to you